github/ci/prow-sig-scale-workloads/roles/bootstrap/tasks/main.yml

```diff
@@ -0,0 +1,133 @@
+---
+- name: disable swap
+  block:
+    - name: swapoff
+      shell: |
+        swapoff -a
+    - name: comment swap entry from fstab
+      replace:
+        path: /etc/fstab
+        regexp: '(.*swap.*)'
+        replace: '#\1'
+    - name: mask systemctl swap unit
+      systemd:
+        name: dev-sda3.swap
+        masked: yes
+
+- name: enable mirrors for yum repos
```

Comment:
This was needed in the `prow-workloads` bare metals, maybe the new nodes have a different setup, have you checked if these changes are required? You can verify by trying to install any package from the configured repos.

Reply:
I needed to install `yum install epel-release`
So, that is why I kept this part, the BM also need some repo configuration